{{ cookiecutter.package_name|slugify }}/Dockerfile

```diff
@@ -66,7 +66,7 @@ RUN --mount=type=cache,target=/root/.cache/ \
 FROM $APP_BASE_IMAGE AS app
 
 # Copy the package source code to the working directory.
-COPY . .
```

Comment:
`pyproject.toml` should already be there because `app` continues from the `ci` image which copies that file, so should be OK.

But not copying everything can be more fragile indeed, because we may forget to copy something that is important. The change here copies the `src/` directory and any `.py` files in the project root. Normally that should be everything you need.